pyphare/pyphare/pharesee/particles.py

```diff
@@ -39,8 +39,6 @@ def __init__(self, **kwargs):
 
 
     def xyz(self, i=0):
-        if self.ndim == 1:
```

Comment:
is this no longer necessary since #552  ?

Reply:
I dont think so... and ih kept, it gives me problems... since now, quantities like `deltas`, `iCells` of a particles are of shape `(numOfPart, ndim)`